ui/job-view/details/tabs/autoclassify/AutoclassifyToolbar.jsx

```diff
@@ -26,10 +26,19 @@ export default class AutoclassifyToolbar extends React.Component {
 
     return (
       <div className="autoclassify-toolbar th-context-navbar navbar-right">
-        {status === 'ready' && <div>
-          {autoclassifyStatus === 'cross_referenced' && <span>Autoclassification pending</span>}
-          {autoclassifyStatus === 'failed' && <span>Autoclassification failed</span>}
-        </div>}
+        {
+          // TODO: This is broken (bug 1504711)
```

Comment:
tbh: I'm not certain what the fix is for this.  I suppose we'll have to cross this bridge when we address re-enabling the autoclassify tab...